cmd/aperture-agent/agent/otel-config.go

```diff
@@ -38,6 +38,7 @@ type AgentOTELConfig struct {
 	BatchPostrollup BatchPostrollupConfig `json:"batch_postrollup"`
 	// CustomMetrics configures custom metrics pipelines, which will send data to
 	// the controller prometheus.
+	// By default `kubeletstats` custom metrics is added, which can be overwritten.
 	CustomMetrics map[string]CustomMetricsConfig `json:"custom_metrics,omitempty"`
```

Comment:
The key of `CustomMetrics` map refers to a pipeline name not metric name. Should clarify this in documentation as well as variable names in the code

Reply:
Done.